gh-pages/vendor/wikia-style-guide/dist/css/lib/components/block-grid.css

```diff
@@ -27,7 +27,9 @@ meta.foundation-data-attribute-namespace {
 html, body {
```

Comment:
You can ignore the css changes till `gh-pages/vendor/wikia-style-guide/dist/vendor/webcomponentsjs/CustomElements.js`, which is a library addition.